physics/GFS_surface_composites_pre.F90

```diff
@@ -241,17 +242,20 @@ subroutine GFS_surface_composites_pre_run (im, lkm, frac_grid,
 
 ! to prepare to separate lake from ocean under water category
       do i = 1, im
-        if ((wet(i) .or. icy(i)) .and. lakefrac(i) > zero) then
-          lake(i) = .true.
-          if (lkm == 1 .and. lakefrac(i) >= 0.15 .and. lakedepth(i) > one) then
-            use_flake(i) = .true.
-          else
-            use_flake(i) = .false.
-          endif
-        else
-          lake(i) = .false.
-          use_flake(i) = .false.
-        endif
+         if( lakefrac(i) > zero .and. lakedepth(i) > one) then
+            lake(i) = .true.
```

Comment:
It seems to me that the flag lake is not used in the code, we can remove this flag.